### ISSUE TITLE
Can http-client.ts be customized?

### ISSUE THREAD

**Issue by DominiYang** [acacode/swagger-typescript-api#700]
Can http-client.ts be customized? Because wechat mini programs or other projects may not be axios or fetch.

I want to wrap a wx.request around it.

**Comment by woutervanvliet** [comment]
Yes it can, quite easily. Use `swagger-typescript-api generate-templates` to get all the templates, and then use `http-client.ejs` to customize `http-client.ts`

**Comment by gaowujian** [comment]
![image](https://github.com/acacode/swagger-typescript-api/assets/29998334/ebc8a7c7-8253-490e-bf18-c6ab695f7140)
I use nodejs scripts to create ts files. you need to create your file structure like this and name the client file as http-client.ejs

**Comment by NicolasGorga** [comment]
Hey @gaowujian I tried what you mention, but no matter what i do, my changes don't get picked up
